src/cljs/pyregence/pages/register.cljs

```diff
@@ -66,18 +67,29 @@
   "The root component for the /register page.
    Displays the register form and a link to the login page."
   [_]
-  [:<>
-   [:div {:style ($/combine ($/disabled-group @pending?)
-                            {:display "flex" :justify-content "center" :margin "5rem"})}
-    [simple-form
-     "Register"
-     "Register"
-     [["Email"            email       "email"    "email"]
-      ["Full Name"        full-name   "text"     "name"]
-      ["Password"         password    "password" "new-password"]
-      ["Confirm Password" re-password "password" "confirm-password"]]
-     register!]]
-   [:div {:style ($/align "flex" "center")}
-    "Already have an account?  "
-    [:a {:href "/login" :style {:margin-left "0.2rem"}} "Log in here"]
-    "."]])
+  (r/create-class
+   {:component-did-mount
+    (fn [_]
+      (let [update-fn (fn [& _]
```

Comment:
Listeners not removed on unmount (same as login),  wrap with `(let [update-fn (atom nil)] ...)` and add:

```clj
  :component-will-unmount
  (fn [_]
    (.removeEventListener js/window "touchend" @update-fn)
    (.removeEventListener js/window "resize" @update-fn))
```